system-addon/content-src/components/TopSites/TopSites.jsx

```diff
@@ -39,15 +48,21 @@ class TopSite extends React.Component {
           visible={isContextMenuOpen}
           onUpdate={val => this.setState({showContextMenu: val})}
           site={link}
-          index={index} />
+          index={index}
+          source={TOP_SITES_SOURCE} />
     </li>);
   }
 }
 
 const TopSites = props => (<section>
   <h3 className="section-title"><FormattedMessage id="header_top_sites" /></h3>
   <ul className="top-sites-list">
-    {props.TopSites.rows.map((link, index) => <TopSite dispatch={props.dispatch} key={link.url} link={link} index={index} />)}
+    {props.TopSites.rows.map((link, index) => <TopSite
+      key={link.url}
+      dispatch={props.dispatch}
+      link={link}
+      index={index}
+      onClickLink={() => this.trackClick(index)} />)}
```

Comment:
why do we need to this track click here as well as in the individual TopSite item?